### DISCUSSION TITLE
Pad string with characters

### DISCUSSION THREAD

**Discussion by Otiel** [mockoon/mockoon#850] [Help] [answered]
I would like to pad a string (left or right) with another character.

Particularly, I'd like to generate identifiers starting with zeros using `{{@index}}` in a `{{#repeat }}` loop: instead of having the following ids : `1, 2, 3, 4, 5, ...`, I'd like to get `00001, 00002, 00003, 00004, 00005, ...`

Is this possible with mockoon helpers or faker.js?

**Comment by 255kb** [comment, marked as answer]
I just looked at Faker documentation and there is no "pad" function. 
We would have to add a `padLeft` helper for this.

**Reply by Otiel** [reply]
Do you want me to create an issue for a feature request?

**Reply by 255kb** [reply]
Yes, you can. Thank you

**Reply by Otiel** [reply]
👍  https://github.com/mockoon/mockoon/issues/855
